cmd/mayactl/app/command/snapshot/create.go

```diff
@@ -35,25 +35,6 @@ $ mayactl snapshot create --volname <vol> --snapname <snap>
 `
 )
 
-/*func init() {
-	host := os.Getenv("MAPI_ADDR")
```

Comment:
Is this code will not be used later ? Please verify with @prateekpandey14

Reply:
i think i had discussed that